release.json

```diff
@@ -13,6 +13,20 @@
 		"JMXFETCH_VERSION": "0.33.1",
 		"JMXFETCH_HASH": "140dd8568b78f5a616da8664f34bb37397c3d3ed87af106d632714f7218e8e88"
 	},
+	"7.16.0-rc.3": {
+		"INTEGRATIONS_CORE_VERSION": "7.16.0-rc.3",
+		"OMNIBUS_SOFTWARE_VERSION": "7.16.0-rc.3",
+		"OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+		"JMXFETCH_VERSION": "0.33.1",
+		"JMXFETCH_HASH": "140dd8568b78f5a616da8664f34bb37397c3d3ed87af106d632714f7218e8e88"
+	},
+	"6.16.0-rc.3": {
+		"INTEGRATIONS_CORE_VERSION": "7.16.0-rc.3",
```

Comment:
Sanity check: we want to use `7.16.0-rc.3` even for `6.16.0-rc.3`?

Reply:
I used `7.16.0-rc.3` because of https://github.com/DataDog/datadog-agent/pull/4535 `LGTM. We should start using 7.x tags on integrations and omnibus, though.`.
@albertvaka : What do you think?

Reply:
Yup, we have a single tag but now it's called `7.something` for both agents instead of `6.something` for both agents.

Reply:
Thanks!